### PR TITLE
update how-to guide to include charting lib info

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: pages/guides/how-to-create-tradingview-charts.mdx [hunk 1]
```diff
@@ -111,9 +111,11 @@ The TradingView integration is a web-based widget that allows users to view Pyth
...
+
 ## Using Datafeed URL with Charting Library
 
-We also provide a datafeed URL that follows the [TradingView Data Integration spec](https://www.tradingview.com/rest-api-spec/#tag/Data-Integration). You can implement your own datafeed utilizing the API or use the built-in [UDF](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF) adapter. If you need a step-by-step guide, refer to the [How to connect data via Datafeed API](https://www.tradingview.com/charting-library-docs/latest/tutorials/implement_datafeed_tutorial/) tutorial.
+We also provide a UDF-compatible URL that follows the [TradingView UDF spec](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF). You can implement your own datafeed utilizing the API or use the built-in [UDF](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF) adapter with the API. If you need a step-by-step guide, refer to the [How to connect data via Datafeed API](https://www.tradingview.com/charting-library-docs/latest/tutorials/implement_datafeed_tutorial/) tutorial, or you can reference the example [here](https://github.com/cctdaniel/pyth-tv-charting-lib)
```
Comment (jayantk) [inline pages/guides/how-to-create-tradingview-charts.mdx:118]: same url is linked twice for UDF. suggest dropping the second one.

File: pages/guides/how-to-create-tradingview-charts.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Tab, Tabs } from "nextra-theme-docs";
 
 The TradingView integration is a web-based widget that allows users to view Pyth prices on their own website. All Pyth prices made available through the TradingView integration are originating from [Pythnet](pythnet-price-feeds.md).
 
-## Technical Integration
```
Comment (jayantk) [inline pages/guides/how-to-create-tradingview-charts.mdx:7]: I suggest explaining in a 2nd paragraph in the info that there are 2 ways to integrate with tradingview, and give the rough tradeoffs between the two there.

File: pages/guides/how-to-create-tradingview-charts.mdx (hunk 1 above)
Comment (jayantk) [reply pages/guides/how-to-create-tradingview-charts.mdx:118]: fwiw the tradingview tutorial above is long and hard to follow. It doesn't give the reader the impression that they can copy/paste some code and get going fast. This doesn't need to be fixed in this PR, but we may want to make our own tutorial that's specific to the Pyth integration.

(i think you have the code here https://github.com/cctdaniel/pyth-tv-charting-lib/blob/main/src/utils/datafeed.js , and then people just need to copy that, plus the tradingview chart around it)

**Review by jayantk — APPROVED** [review]

File: pages/guides/how-to-create-tradingview-charts.mdx (hunk 1 above)
Comment (cctdaniel) [reply pages/guides/how-to-create-tradingview-charts.mdx]: by "tutorial" do you mean literally a step-by-step example? the repo that I've linked to is the "tutorial" that I was thinking of where users can just copy the files and get it working, maybe I should expand more on that to make it clearer to the users?

**Review by jayantk — APPROVED** [review]
